Source/Core/Matrix4.js

```diff
@@ -675,9 +643,7 @@ define([
         if (far <= 0.0) {
```

Comment:
Can these be checked with `Check.numeric.minimum` and `maximum`? I'm not sure what the best way to handle `<=` is though, something like `Check.numeric.minimum(CesiumMath.EPSILON7)` would be clunky.